src/mlpack/tests/logistic_regression_test.cpp

```diff
@@ -836,7 +859,7 @@ BOOST_AUTO_TEST_CASE(ClassifyTest)
 
   // Now train a logistic regression object on it.
   LogisticRegression<> lr(data.n_rows, 0.5);
-  lr.Train<>(data, responses);
+  lr.Train<> (data, responses);
```

Comment:
```suggestion
  lr.Train<>(data, responses);
```

That should resolve the static analysis issue.

Reply:
Yup that solved it.